<ui version="4.0" >
 <class>AddLinkForm</class>
 <widget class="QDialog" name="AddLinkForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>441</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Visual Netkit - Add link</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>The Future Link</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>3</number>
      </property>
      <property name="leftMargin" >
       <number>2</number>
      </property>
      <property name="topMargin" >
       <number>2</number>
      </property>
      <property name="rightMargin" >
       <number>2</number>
      </property>
      <property name="bottomMargin" >
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="labelVm" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection" >
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line" >
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth" >
           <number>4</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelCd" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor" >
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="scaledContents" >
           <bool>false</bool>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::NoTextInteraction|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Hardware Interface</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Interface name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="ethNumberSpinBox" >
          <property name="prefix" >
           <string>eth</string>
          </property>
          <property name="maximum" >
           <number>128</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Interace state: </string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ethUp" >
          <property name="text" >
           <string>Up</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ethDown" >
          <property name="text" >
           <string>Down</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Existing interfaces</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QTableWidget" name="interfacesTable" >
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>Interface Name</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="addLinkButtonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addLinkButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddLinkForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
